src/components/Choropleth.vue

```diff
@@ -48,15 +52,20 @@ const featureInSelectedCountry = (feature: Feature, selectedCountry: string) =>
     feature.properties[featureProperties.country] === selectedCountry;
 
 const getFeatureId = (feature: Feature) =>
-    featureInSelectedCountry(feature, selectedCountryId.value)
+    featureInSelectedCountry(feature, selectedCountryId.value) && selectedAdminLevel.value !== 1
```

Comment:
this would give you the admin level 2 feature ids when admin level 1 is selected, but we want admin 1 ids when it is selected otherwise the tooltips would be wrong